physics/cu_gf_driver_post.meta

```diff
@@ -83,6 +83,34 @@
   type = real
   kind = kind_phys
   intent = out
+[rrfs_sd]
+  standard_name = do_smoke_coupling
+  long_name = flag controlling rrfs_sd collection (default off)
```

Comment:
@mkavulich Updated. Thanks.

Reply:
@mkavulich Long names are not currently being parsed in the CCPP, so we have been allowing deviations that clarify the variable because it is more-or-less serving as documentation. AFAIK, there isn't a requirement that long names be consistent across hosts and schemes.

Reply:
> @mkavulich Long names are not currently being parsed in the CCPP, so we have been allowing deviations that clarify the variable because it is more-or-less serving as documentation. AFAIK, there isn't a requirement that long names be consistent across hosts and schemes.

This is true for both prebuild and capgen.